actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -480,6 +487,8 @@ def []=(name, options)
           options = { value: options }
         end
 
+        preprocess_options(options)
```

Comment:
Two lines after this the call `@parent_jar[name]` will lead to `preprocess_options` being invoked in the `@parent_jar`. Thus, I think this call maybe unnecessary...